src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -216,4 +219,57 @@ public static UserType parseUserType(String userType) throws ParseException {
         return new UserType(trimmedUserType);
     }
 
+    /**
+     * Parses a {@code Collection<String> userImage} into a {@code Set<UserImage>}.
+     */
+    public static Set<UserImage> parseUserImages(Collection<String> userImages) throws ParseException {
+        requireNonNull(userImages);
+        Set<UserImage> userImageSet = new LinkedHashSet<>();
+        for (String userImage : userImages) {
+            userImageSet.add(parseUserImage(userImage));
+        }
+        return userImageSet;
+    }
+
+    /**
+     * Parses a {@code String userImage} into a {@code UserImage}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code userImage} is invalid.
+     */
+    public static UserImage parseUserImage(String userImage) throws ParseException {
+        String trimmedUserImage = userImage.trim();
+        String[] splitUserImage = trimmedUserImage.split(":");
+
+        FilePath filePath = parseFilePath(splitUserImage[0]);
+        String description = "";
+        if (splitUserImage.length == 2) {
+            description = parseDescription(splitUserImage[1]);
+        }
+        return new UserImage(filePath, description);
+    }
+
+    /**
+     * Parses a {@code String filePath} into a {@code FilePath}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code filePath} is invalid.
+     */
+    public static FilePath parseFilePath(String filePath) throws ParseException {
+        requireNonNull(filePath);
+        String trimmedFilePath = filePath.trim();
+        if (!FilePath.isValidFilePath(trimmedFilePath)) {
+            throw new ParseException(FilePath.MESSAGE_CONSTRAINTS);
+        }
+        return new FilePath(trimmedFilePath);
+    }
+
+    /**
+     * Leading and trailing whitespaces will be trimmed.
+     */
```

Comment:
Looks like an incomplete javadoc

Reply:
Thanks for catching. Javadoc has been updated.
```
   /**
     * Parses a {@code String Description} into a {@code description}.
     * Leading and trailing whitespaces will be trimmed.
     * /
```